changelog/19.0/19.0.0/summary.md

```diff
@@ -19,6 +19,8 @@
     - [`SHOW VSCHEMA KEYSPACES` Query](#show-vschema-keyspaces)
   - **[Planned Reparent Shard](#planned-reparent-shard)**
     - [`--tolerable-replication-lag` Sub-flag](#tolerable-repl-lag)
+  - **[Apply VSchema](#apply-vschema)**
+    - [`--strict` sub-flag and `strict` gRPC field](#strict-flag-and-field)
```

Comment:
I don't think that this is a major change and needs to be here. That being said, I'm also fine leaving it here.

Reply:
I'll create a new `Minor` section and move it there

Reply:
I'm not against that really, but at last in my view the summary is for major changes that every Vitess user should be aware of. Typically a new flag like this is not included.

Reply:
Went ahead and added to minor section for now